third_party/terraform/tests/resource_compute_network_endpoint_test.go.erb

```diff
@@ -42,18 +39,32 @@ func TestAccComputeNetworkEndpoint_networkEndpointsBasic(t *testing.T) {
 				// Force-recreate old endpoint
 				Config: testAccComputeNetworkEndpoint_networkEndpointsModified(context),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeNetworkEndpointWithPortExists("google_compute_network_endpoint.default", "100"),
 					testAccCheckComputeNetworkEndpointWithPortsDestroyed(negId, "90"),
```

Comment:
I'm assuming we're keeping this because it's testing behaviour that import-style tests can't check?

Reply:
Yep, as far as I can tell. I err on the side of caution with checks that look like they are doing something slightly more. I can double check, and get it on another import-style sweep.

Reply:
Sounds good! Just wanted to make sure we were on the same page.